MAINTAINERS

```diff
@@ -22,6 +22,7 @@
 #
 #Maintainers 
 "Amit Kumar Das",@AmitKumarDas,JIMDO
+"Giridhar Shankar",@giridharshankar013,Datacore
```

Comment:
@pawanpraka1 - can you please update this along with if there are any further changes required here?